pages/vi/gitrepositories.md

```diff
@@ -79,7 +79,7 @@ This is just a summary of the steps that you will need to perform. Keep on readi
 
 ### Clone Your GitHub Repository username.github.io
 
-Now, we will be using GitHub repositories on a command line, which means that there is a separate step to get your GitHub repository on your OS. To be clear, you will be using both the command line and the GitHub user interface, meaning that you **need to constantly check to make sure that your version is not behind to avoid merge conflicts**. Therefore, open your username.github.io repository on the GitHub user interface and copy the link provided by clicking the "Clone or download" button.
+Now, we will be using GitHub repositories on a command line, which means that there is a separate step to get your GitHub repository on your OS. To be clear, you will be using both the command line and the GitHub user interface, meaning that you **need to constantly check to make sure that your version is not behind to avoid merge conflicts**. Therefore, open your username.github.io repository on the GitHub user interface and copy the link provided by clicking the "Code" button.
```

Comment:
I noticed that this change was carried over from this other pull request https://github.com/treehouses/treehouses.github.io/pull/812. Would you mind redoing this commit so that it doesn't have this file changed and thus, only has the change related the wifi-adaptor issue.

Reply:
Oh no. 😔 I reverted the accidental file changes. Is this how it's done on GitHub, or will I need to do something else like create another pull request?